hari_client/client/client.py

```diff
@@ -1330,7 +1330,8 @@ def create_media_objects(
 
         # 1. parse media_objects to dicts before upload
         media_object_dicts = [
-            media_object.model_dump() for media_object in media_objects
+            media_object.model_dump(exclude={"uploaded", "id"})
```

Comment:
I think you already exclude it with a pydantic
```suggestion
            media_object.model_dump(exclude={"uploaded"})
```